src/main/resources/messages/LibertyBundles.properties

```diff
@@ -134,4 +134,5 @@ gradle.wrapper.does.not.exist=A Gradle wrapper for the project could not be foun
 gradle.wrapper.cannot.execute=Could not execute Gradle wrapper because the process does not have permission to execute it. Consider giving executable permission for the Gradle wrapper file or changing the build preferences for Gradle inside IntelliJ Gradle preferences.
 gradle.invalid.build.preference=Make sure to configure a valid path for Gradle inside IntelliJ Gradle preferences.
 gradle.cannot.execute=Could not execute Gradle from {0} because the process does not have permission to execute it. Consider giving executable permission for the Gradle executable or configure IntelliJ to use the Gradle wrapper.
-gradle.does.not.exist=Could not execute the Gradle executable {0}. Make sure a valid path is configured inside IntelliJ Gradle preferences.
\ No newline at end of file
+gradle.does.not.exist=Could not execute the Gradle executable {0}. Make sure a valid path is configured inside IntelliJ Gradle preferences.
+gradle.settings.is.null=Cannot execute project because there is an error with configured Gradle. Make sure to configure a valid path for Gradle inside IntelliJ Gradle preferences.
```

Comment:
```suggestion
gradle.settings.is.null=Could not execute action because there is an error with Gradle configuration. Make sure to configure a valid path for Gradle inside IntelliJ Gradle preferences.
```